plugins/api/src/atomic_apply_acl_operations.cpp

```diff
@@ -378,7 +379,11 @@ namespace
             throw_if_invalid_acl(acl);
 
             log::api::trace("Retrieving entity ID ...");
-            const auto entity_id = get_entity_id(_db_conn, _op.at("entity_name").get<std::string>());
+            auto zone = std::string{getLocalZoneName()};
+            if(_op.find("zone_name") != std::end(_op)) {
+                zone = _op.at("zone_name").get<std::string>();
+            }
+            const auto entity_id = get_entity_id(_db_conn, _op.at("entity_name").get<std::string>(), zone);
```

Comment:
`.get` -> `.get_ref` ?

Reply:
Since I switched to `std::string_view` above, `.get_ref` doesn't work anymore.

Reply:
Check the API docs for nlohmann JSON to see if string_view is acceptable. If you can't find any documentation for it, open an issue in the nlohmann JSON github repo about using string_view with `.get` and `.get_ref`.